src/tests/frontend/helper.js

```diff
@@ -185,7 +180,7 @@ const helper = {};
     // set new iframe
     $('#iframe-container').append($iframe);
     $iframe.one('load', () => {
-      helper.admin$ = getFrameJQuery($('#iframe-container iframe'));
+      helper.admin$ = getFrameJQuery($('#iframe-container iframe'), false, false);
```

Comment:
Why shouldn't it load jQuery here?

Reply:
not needed, as it's already src'ed in a script tag. However, to be sure I changed it to "true, false" so that it will check if win.$ is present and download jquery if not.